ntc_templates/templates/mikrotik_routeros_interface_print_detail.textfsm

```diff
@@ -3,27 +3,30 @@ Value DYNAMIC (D)
 Value STATUS (X|R)
 Value SLAVE (S)
 Value NAME (\S+)
-Value List DESCRIPTION ((?!\s*$).+[^\s])
+Value List DESCRIPTION (.+)
```

Comment:
```suggestion
Value List DESCRIPTION (.+?)
```

avoid accidently grabbing spaces

Reply:
The suggested change causes multiline comments to not be parsed

![image](https://github.com/user-attachments/assets/5f422a0b-fb92-4e48-a21f-083adb8b3c4a)

Reply:
That looks to do exactly what I wanted it to do. As the template currently stands, it is capturing trailing space, but with the `?` it is capturing only up to the trailing space.

Reply:
actually, looking at that example, we also want to put a `\s*` in front of the description capture to avoid catching any leading whitespace

Reply:
Oh yes I see. I'll make those changes and update the test files and commit them.